guides/orders/order-state-machine.md

```diff
@@ -16,12 +16,8 @@ There are six order states by default:
 If you go through the checkout provided in the `solidus_frontend` gem, you can
 see that there is a clearly defined step for each of these states during checkout.
 
-In some circumstances, the `payment` and `confirm` states are optional:
-
-- The `payment` state is only triggered if `payment_required?` returns `true`.
-- The `confirm` state is only triggered if `confirmation_required?` returns
-  `true`.
-
+The `payment` state is optional and it's only triggered if `payment_required?` 
+returns `true`.
```

Comment:
We should add a newline after this paragraph to separate it from the paragraph below.

Reply:
Ok! I thought that since it's a single line it would be better to compact those paragraphs but here is your land, I will update shortly! 😄

Reply:
done!